login.html

```diff
@@ -91,11 +108,14 @@
 <div>
 <input type="password" name="password" required placeholder="Password">
 </div>
+<div class="fp">
+<a class="a" href="login.html">Forget Password</a>
```

Comment:
move Forget Password under sign up and reset not above them

Reply:
ok